<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1184</width>
    <height>924</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="nrOfMarkedFrames">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>540</y>
      <width>99</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>nrOfMarkedFrames</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>590</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Chapter</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Frames</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>New Item</string>
     </property>
     <item>
      <property name="text">
       <string>New Subitem</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>New Item</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>New Subitem</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </item>
    </item>
   </widget>
   <widget class="QFrame" name="videoFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>441</width>
      <height>271</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>567</width>
      <height>36</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonStop">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPlayPause">
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStep">
       <property name="text">
        <string>Step</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonMark">
       <property name="text">
        <string>Mark</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="videoName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>961</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Video name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTest">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>470</y>
      <width>125</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Test mis</string>
    </property>
   </widget>
   <widget class="QListWidget" name="marksListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>541</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Marked frames:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDebugInfo">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>540</y>
      <width>441</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Debug info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSave">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>320</y>
      <width>125</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>330</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Preview:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelImageGeometry">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>330</y>
      <width>99</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMontageArea">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>431</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="sliderVideoPosition">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>451</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>520</y>
      <width>211</width>
      <height>36</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonDeleteMark">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClearMarks">
       <property name="text">
        <string>Clear all</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>431</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="currentTime">
       <property name="text">
        <string>currentTime</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="currentTimeMs">
       <property name="text">
        <string>currentMs</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="videoDuration">
       <property name="text">
        <string>duration</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1184</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
  </widget>
  <action name="actionOpen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
